felix/bpf/ut/icmp_test.go

```diff
@@ -79,7 +79,7 @@ func TestICMPCTPlain(t *testing.T) {
 			Expect(k.PortB()).To(Equal(uint16(0)))
 		}
 	})
-	expectMark(tcdefs.MarkSeen)
+	expectMark(tcdefs.MarkSeenSkipFIB)
```

Comment:
I do not see how the change should modify this. We may have unclean conntrack at the start of this test or there is a bug.

Reply:
Fixed. It happened because `CALI_SKB_MARK_SEEN & CALI_SKB_MARK_SKIP_FIB` is equal to `CALI_SKB_MARK_SEEN` which made the whole condition True. It's fixed now.